.circleci/config.yml

```diff
@@ -71,28 +66,43 @@ commands:
           command: |
             curl -fLSs \
             https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh | sudo bash
+  quality:
+    description: "Run quality"
+    steps:
+      - set_up_environment
+      - run_with_languages:
+          label: Run quality
+          command: |
+            bundle check # TODO
```

Comment:
[punchlist] Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker